object-browser-schema.js

```diff
@@ -0,0 +1,199 @@
+export default {
+  additionalProperties: false,
+  definitions: {
+    actionItemDisabled: {
+      description: 'Tells if action bar button or link is disabled or ties it to selection',
+      oneOf: [
+        {
+          type: 'boolean',
+          default: false,
+        }, {
+          type: 'object',
+          additionalProperties: false,
+          description: 'Shortcut to enable and disable button or link as selection changes',
+          properties: {
+            minSelected: {type: 'number'},
+            maxSelected: {type: 'number'}
+          }
+        }
+      ]
+    },
+    actionItem: {
+      type: 'object',
+      properties: {
+        id: {
+          type: 'string',
+          description: 'Given to action handler to identify button or link'
+        },
+        text: {
+          type: 'string',
+          description: 'The text displayed on button or link'
+        },
+        size: {
+          type: 'string',
+          description: 'Size of button or link',
+          enum: ['small', 'medium', 'large'],
+          default: 'small'
+        },
+        priority: {
+          type: 'string',
+          description: 'Priority of button or link',
+          enum: ['primary', 'secondary', 'tertiary'],
+          default: 'primary'
+        },
+        disabled: {
+          '$ref': '#/definitions/actionItemDisabled'
+        }
+      }
+    }
+  },
+  type: 'object',
+  description: 'Object browser configuration schema',
+  properties: {
+    bunsenModel: {
+      type: 'object',
+      description: 'Bunsen model for list items'
+    },
+    bunsenView: {
+      ref: 'https://github.com/ciena-blueplanet/bunsen-core/blob/master/src/validator/view-schemas/v2.js',
+      description: 'Bunsen view for list items'
+    },
+    expandable: {
+      type: 'boolean',
+      default: false,
+      description: 'Is list expandable?'
+    },
+    isExpanded: {
+      type: 'boolean',
+      default: false,
+      description: 'Is list currently expanded'
+    },
+    rowHeight: {
+      type: 'number',
+      description: 'Row height given to frost-list. Default is frost-list default.'
```

Comment:
Shouldn't be necessary, we're not basing scroll behavior on the row height (set to variable)